structure/segmenttree/segmenttree.go

```diff
@@ -113,6 +113,11 @@ func (s *SegmentTree) Build(node int, left int, right int) {
 	}
 }
 
+// Returns a new instance of a SegmentTree with lazy propagation,
```

Comment:
Please follow the [Go Doc comments](https://tip.golang.org/doc/comment) style of commenting 🙏🏼